compiler/parser/Cargo.toml

```diff
@@ -14,15 +14,15 @@ serde = ["dep:serde", "rustpython-compiler-core/serde"]
 
 [build-dependencies]
 anyhow = { workspace = true }
-lalrpop = { version = "0.19.9", optional = true }
+lalrpop = { version = "0.19.10", optional = true }
```

Comment:
lalrpop 0.19.10 has a bug. idk if it affect rustpython.

Reply:
Yeah, let me revert this change. It's unrelated to the other changes